fusion-cli/build/get-webpack-config.js

```diff
@@ -398,12 +395,29 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
       runtime === 'server' &&
         ((context, request, callback) => {
           if (/^[@a-z\-0-9]+/.test(request)) {
-            // do not bundle external packages and those not whitelisted
             const absolutePath = resolveFrom.silent(context, request);
+            // do not bundle external packages and those not whitelisted
             if (absolutePath === null) {
               // if module is missing, skip rewriting to absolute path
               return callback(null, request);
             }
+            if (experimentalCompileTest) {
+              const {bundle} = experimentalCompileTest(absolutePath, {
+                transform: 'none', // default transform value doesn't actually matter here
```

Comment:
```suggestion
                transform: 'spec',
```

For browser-only bundling, the default option should be `transform: spec`, right?

Reply:
Technically this is being called to decide whether to bundle a external dependency or not, so the transpile default for external dependencies is none, since they are not bundled. Also, we don't care about the transform response here so it can technically be anything

Reply:
Oh right, the externals function only gets called for bare imports.